.circleci/config.yml

```diff
@@ -1,3 +1,5 @@
+# This file is automatically transpiled to config.yml by the transpileCircleCiConfigTask
+# during any ./gradlew task. ONLY edit this .kson file - the .yml file will be overridden.
```

Comment:
Hmmm interesting. This comes from the original KSON file, I guess, but it's confusing. Maybe the gradle task should prepend a line saying something like `# This file has been automatically generated from config.kson`